main.cc

```diff
@@ -7,7 +7,7 @@
 int main() {
 
   dev::Calculator c;
-  std::cout << c.calc("((2.5 + 2.5) * 9.0) / 3.0") << std::endl;
+  std::cout << c.calc("4 12 3 + * 2 / 5 5 + * 100 2 * - 2 /") << std::endl;
```

Comment:
Suppose that you using main to check is program is working. Suggest to use unit tests for it

Reply:
Done.